protocols/imap/src/main/java/org/apache/james/imap/decode/ImapRequestLineReader.java

```diff
@@ -372,7 +372,7 @@ public String consumeLiteral(Charset charset) throws DecodingException {
             return consumeLiteral(US_ASCII);
         } else {
             try {
-                ImmutablePair<Integer, Literal> literal = consumeLiteral(false);
+                ImmutablePair<Integer, Literal> literal = consumeLiteral(true);
```

Comment:
I'm not sure how this change relates to the rest ...

Reply:
Crap. You are fully right.